zomes/blog/capabilities/hc_lifecycle/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "capability": {
+    "membrane": "zome"
+  },
+
+  "functions": [
+    {
+        "name": "genesis",
```

Comment:
this isn't required by hc currently, `NotImplemented` is a pass

why would i ever want to add an empty genesis, other than to demonstrate that it exists?

Reply:
As discussed today, this is not polished yet and mainly exists as a crystallization point for further discussion.

Imho, if there is a capability that is named `hc_lifecycle`, it should implement lifecycle functions like genesis, because we think of predefined system capabilities like this as traits that a zome can implement.